src/Nethermind/Nethermind.Consensus.AuRa/Validators/AuRaValidatorsCollectionExtensions.cs

```diff
@@ -21,6 +21,9 @@ namespace Nethermind.Consensus.AuRa.Validators
 {
     internal static class AuRaValidatorsCollectionExtensions
     {
-        public static int MinSealersForFinalization(this IList<Address> validators) => validators.Count / 2 + 1;
+        public static int MinSealersForFinalization(this IList<Address> validators, bool twoThirds = false)
+        {
+            return (twoThirds ? validators.Count * 2 / 3 : validators.Count / 2) + 1;
```

Comment:
seems incorrect - for 5 validators it will tell that 3 is enough to seal and you need 4 2/3 * 5 is 3.3

Reply:
aaah, there is + 1 there

Reply:
hmmmm but then for 6 it will tell 5 and it should tell 4

Reply:
we need tests for various numbers

Reply:
Its > 2/3 of validators, not >= 2/3 validators.

So for 6, 2/3 validators = 4, so its > 4, then its 5

Reply:
will it then fail for one validator? 1 * 2 / 3 == 1, then + 1 will return 2

Reply:
for 7 it will return 6 and should return 5

Reply:
for any 3*n + 1 it will return wrong number